dallinger/command_line.py

```diff
@@ -553,6 +591,7 @@ def _deploy_in_mode(mode, app, verbose):
 @dallinger.command()
 @click.option('--verbose', is_flag=True, flag_value=True, help='Verbose mode')
 @click.option('--app', default=None, help='Experiment id')
+@timeout(21600)
```

Comment:
I'm not sure "timeout" is the right name, since it doesn't really time anything out. "report_idle_after"?